packages/waku/src/lib/renderers/dev-worker-impl.ts

```diff
@@ -73,8 +89,8 @@ const handleRender = async (mesg: MessageReq & { type: 'render' }) => {
         isDev: true,
         loadServerFile,
         loadServerModule,
-        resolveClientEntry: (id: string) =>
-          resolveClientEntryForDev(id, rest.config),
+        resolveClientEntry: (file: string) =>
+          resolveClientEntryForDev(file, rest.config, initialModules),
```

Comment:
If renaming to `file` is intentional,

Reply:
changed it for debugging, now i reverted it.